benchmarks/Schelling/schelling.py

```diff
@@ -16,23 +18,22 @@ def __init__(self, unique_id, model, agent_type):
         """
         super().__init__(unique_id, model)
         self.type = agent_type
+        self.radius = radius
```

Comment:
Why was radius moved from the model to the agent?

Reply:
Because it is an attribute of the agent. Any time you start repeatedly doing in an agent self.model.{some_attribute}, you should just move it to the agent itself.